3rdparty/indi-atik/atik_ccd.cpp

```diff
@@ -357,8 +357,10 @@ bool ATIKCCD::setupParams()
     PrimaryCCD.setMinMaxStep("CCD_BINNING", "HOR_BIN", 1, binX, 1, false);
     PrimaryCCD.setMinMaxStep("CCD_BINNING", "VER_BIN", 1, binY, 1, false);
 
-    IUSaveText(&VersionInfoS[VERSION_FIRMWARE], std::to_string(pProp.Protocol).c_str());
-    LOGF_INFO("Detected camera %s %s with firmware %s", pProp.Manufacturer, pProp.Description, std::to_string(pProp.Protocol).c_str());
+    char firmware[8] = {0};
+    snprintf(firmware, sizeof(firmware), "%d.%d", prop.Protocol >> 8, prop.Protocol & 0xff);
```

Comment:
Doesn't compile like this, "prop" here should be "pProp".

Reply:
Thanks, I fixed it in [PR](https://github.com/indilib/indi/pull/976)

There are additional compile errors when compiling with g++ (Debian 8.3.0-6) 8.3.0
under Debian Buster:

/home/tstibor/dev/astronomy/indi/3rdparty/indi-atik/atik_ccd.cpp: In member function ‘bool ATIKCCD::grabImage()’:
/home/tstibor/dev/astronomy/indi/3rdparty/indi-atik/atik_ccd.cpp:844:10: error: ‘unique_lock’ is not a member of ‘std’
     std::unique_lock<std::mutex> guard(ccdBufferLock);
          ^~~~~~~~~~~
/home/tstibor/dev/astronomy/indi/3rdparty/indi-atik/atik_ccd.cpp:844:10: note: ‘std::unique_lock’ is defined in header ‘<mutex>’; did you forget to ‘#include <mutex>’?
/home/tstibor/dev/astronomy/indi/3rdparty/indi-atik/atik_ccd.cpp:31:1:
+#include <mutex>
 
/home/tstibor/dev/astronomy/indi/3rdparty/indi-atik/atik_ccd.cpp:844:10:
     std::unique_lock<std::mutex> guard(ccdBufferLock);
          ^~~~~~~~~~~
/home/tstibor/dev/astronomy/indi/3rdparty/indi-atik/atik_ccd.cpp:844:27: error: ‘mutex’ is not a member of ‘std’
     std::unique_lock<std::mutex> guard(ccdBufferLock);
                           ^~~~~
/home/tstibor/dev/astronomy/indi/3rdparty/indi-atik/atik_ccd.cpp:844:27: note: ‘std::mutex’ is defined in header ‘<mutex>’; did you forget to ‘#include <mutex>’?
/home/tstibor/dev/astronomy/indi/3rdparty/indi-atik/atik_ccd.cpp:844:40: error: ‘ccdBufferLock’ was not declared in this scope
     std::unique_lock<std::mutex> guard(ccdBufferLock);
                                        ^~~~~~~~~~~~~
/home/tstibor/dev/astronomy/indi/3rdparty/indi-atik/atik_ccd.cpp:844:34: error: ‘guard’ was not declared in this scope
     std::unique_lock<std::mutex> guard(ccdBufferLock);
                                  ^~~~~
make[2]: *** [CMakeFiles/indi_atik_ccd.dir/build.make:63: CMakeFiles/indi_atik_ccd.dir/atik_ccd.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:110: CMakeFiles/indi_atik_ccd.dir/all] Error 2
make: *** [Makefile:130: all] Error 2

Reply:
The compile problems above are a mistake of my side. I had Debian Buster libindi 1.7.5 installed
and the atik driver includes the dev files from libindi 1.7.5. Cleaning up old libindi and compiling
master again, make it worked:
Scanning dependencies of target indi_atik_wheel
[ 25%] Building CXX object CMakeFiles/indi_atik_wheel.dir/atik_wheel.cpp.o
[ 50%] Linking CXX executable indi_atik_wheel
[ 50%] Built target indi_atik_wheel
Scanning dependencies of target indi_atik_ccd
[ 75%] Building CXX object CMakeFiles/indi_atik_ccd.dir/atik_ccd.cpp.o
[100%] Linking CXX executable indi_atik_ccd
[100%] Built target indi_atik_ccd